store/copr/coprocessor.go

```diff
@@ -954,6 +954,22 @@ func (worker *copIteratorWorker) handleCopResponse(bo *Backoffer, rpcCtx *tikv.R
 		data := make([]byte, len(cacheValue.Data))
 		copy(data, cacheValue.Data)
 		resp.pbResp.Data = data
+		if worker.req.Paging {
+			var start, end []byte
+			if cacheValue.PageStart != nil {
+				start = make([]byte, len(cacheValue.PageStart))
+				copy(start, cacheValue.PageStart)
+			}
+			if cacheValue.PageEnd != nil {
+				end = make([]byte, len(cacheValue.PageEnd))
+				copy(end, cacheValue.PageEnd)
+			}
```

Comment:
Seeing this, I have another idea: we can just remove `PageStart != nil`. Both `make` and `copy` works well for nil value. This could probably be applied to L989, too.

Reply:
But if `cacheValue.PageStart` is nil, after the operation, start become `[]byte{}`
That's not what I want

Reply:
> But if `cacheValue.PageStart` is nil, after the operation, start become `[]byte{}` That's not what I want

Got it.